src/edu/wright/cs/raiderplanner/controller/SettingsController.java

```diff
@@ -0,0 +1,412 @@
+/*
+ * Copyright (C) 2017 - Benjamin Dickson, Andrew Odintsov, Zilvinas Ceikauskas,
```

Comment:
If you created this file, then you should not add the year/names for the original authors of the project. Move yours up to the first line and leave the remaining three or four lines blank,

Reply:
So, it would be:
/*
 * Copyright (C) 2018 - Clayton D. Terrill
 *
 * 
 *
 * This program is free software: you can redistribute it and/or modify
 * it under the terms of the GNU General Public License as published by
 * the Free Software Foundation, either version 3 of the License, or
 * (at your option) any later version.
 *
 * This program is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 * GNU General Public License for more details.
 *
 * You should have received a copy of the GNU General Public License
 * along with this program.  If not, see <http://www.gnu.org/licenses/>.
 *
 */

Reply:
Yes, that is correct.